### PR TITLE
Resolve discussions

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
**What kind of change does this PR introduce?**
Resolve discussions for
- `SSEDownNotice`: #1965
- `GitHub`: #1968
- `More`: #1970

**What is the current behavior?**
N/A

**What is the new behavior?**
N/A

**Checklist**:
- [ ] Documentation N/A
- [x] Tests
- [x] Ready to be merged 
- [ ] Added myself to contributors table  N/A

### CODE REVIEW

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/items/More/More.tsx [hunk 1]
```diff
@@ -36,7 +38,7 @@ export const More = observer(() => {
           <ProgressButton
             block
             loading={isForkingSandbox}
-            onClick={() => forkSandboxClicked()}
+            onClick={forkSandboxClicked}
```
Comment (CompuIves) [inline packages/app/src/app/pages/Sandbox/Editor/Workspace/items/More/More.tsx]: Did you test that this works without warnings? I'm not sure if Cerebral likes receiving an event.

Comment (MichaelDeBoey) [reply packages/app/src/app/pages/Sandbox/Editor/Workspace/items/More/More.tsx]: Not tested manually, but this is just basic JS, no? 🤔

Comment (CompuIves) [reply packages/app/src/app/pages/Sandbox/Editor/Workspace/items/More/More.tsx]: It is, I mean that the signal of Cerebral can receive an object. Cerebral will not execute the signal if that object is not serializable (like functions). I'm not sure if the argument of `onClick` is an event and whether it's serializable. That's probably why I used an arrow func here.

I could also be wrong though.

Comment (MichaelDeBoey) [reply packages/app/src/app/pages/Sandbox/Editor/Workspace/items/More/More.tsx]: Don't know enough of `Cerebral` to know what's happening, but this is the implementation of `forkSandboxClicked`:

```js
export const forceForkSandbox = [
  when(state`editor.currentSandbox.owned`),
  {
    true: [
      actions.confirmForkingOwnSandbox,
      {
        confirmed: forkSandbox,
        cancelled: [],
      },
    ],
    false: forkSandbox,
  },
];
```

Comment (MichaelDeBoey) [reply packages/app/src/app/pages/Sandbox/Editor/Workspace/items/More/More.tsx]: And  this is the used `confirmForkingOwnSandbox`:

```js
export function confirmForkingOwnSandbox({ browser, path }) {
  return browser.confirm('Do you want to fork your own sandbox?')
    ? path.confirmed()
    : path.cancelled();
}
```

Comment (CompuIves) [reply packages/app/src/app/pages/Sandbox/Editor/Workspace/items/More/More.tsx]: That makes sense, the signal executor passes the info. The best way to be sure is by just pressing the fork button at the "More" screen.

Comment (CompuIves) [reply packages/app/src/app/pages/Sandbox/Editor/Workspace/items/More/More.tsx]: I think the PR build is here: https://pr1987.build.csb.dev/s/vanilla. But it seems like there's an error: 
image

(cc @lbogdan)

**Comment by MichaelDeBoey** [comment]
@CompuIves Since all checks pass, I think this one can be merged after a last review 🙂

**Comment by MichaelDeBoey** [comment]
Ping @CompuIves

**Comment by CompuIves** [comment]
Very clean!

**Comment by MichaelDeBoey** [comment]
I try to be a good boy 😇😇

**Pull request merged**